### DISCUSSION TITLE
Paradox Mods Support?

### DISCUSSION THREAD

**Discussion by Randomrandom333** [slyh/Cities2-TrafficLightsEnhancement#90] [Q&A]
I have recently fully switched from using Thunder Store for mods to using Paradox Mods and I am wondering if there are any plans to put this mod on there.  This is one of those mods which is truly a lifesaver when building large cities, and I would love to see it easily accessible from PDX Mods instead of just thunder store.

**Comment by Me1onSeed** [comment]
Please! T_T.

**Reply by OutsideOctaves** [reply]
I'll link to issue https://github.com/slyh/Cities2-TrafficLightsEnhancement/issues/87 ,  Here you can see he is working on it, just hope it drops soon myself, though I'd rather a quality thing that takes a bit more time than a sloppy release now :D
